app/components/CommunityBoardBudgetRequestPanel/index.tsx

```diff
@@ -0,0 +1,114 @@
+import { CommunityBoardBudgetRequest } from "~/gen";
+import {
+  Flex,
+  Heading,
+  HStack,
+  IconButton,
+  Tag,
+  Text,
+  Tooltip,
+  VStack,
+} from "@nycplanning/streetscape";
+import { ChevronLeftIcon, InfoIcon } from "@chakra-ui/icons";
+
+export type CommunityBoardBudgetRequestPanelProps = {
+  cbbr: CommunityBoardBudgetRequest;
+  agencyName: string | undefined;
+  policyArea: string | undefined;
+  agencyResponseType: string | undefined;
+  onNavigationClick: () => void;
+};
+
+export function CommunityBoardBudgetRequestPanel({
+  cbbr,
+  agencyName,
+  policyArea,
+  agencyResponseType,
+  onNavigationClick,
+}: CommunityBoardBudgetRequestPanelProps) {
+  return (
+    <VStack alignItems={"flex-start"} gap={2}>
+      <HStack align={"center"}>
+        <IconButton
+          aria-label="Close budget request detail panel"
+          icon={<ChevronLeftIcon boxSize={6} />}
+          color={"gray.600"}
+          backgroundColor={"inherit"}
+          _hover={{
+            border: "none",
+            backgroundColor: "blackAlpha.100",
+          }}
+          onClick={onNavigationClick}
+        />
+        <Heading color="gray.600" fontWeight={"bold"} fontSize={"md"}>
+          {cbbr.title}
+          {cbbr.isContinuedSupport && "*"}
+        </Heading>
+      </HStack>
+
+      <VStack
+        alignItems={"flex-start"}
+        borderBottom={"1px solid"}
+        borderColor={"gray.200"}
+        width="100%"
+        paddingBottom={2}
+        justifyContent={"flex-start"}
+      >
+        <Flex gap={2} flexWrap={"wrap"}>
+          <HStack>
+            <Text fontWeight={700}>Tracking Number: </Text>
+            <Text>{cbbr.id}</Text>
+          </HStack>
+          <HStack>
+            <Text fontWeight={700}>Community Board: </Text>
+            <Tag>{cbbr.communityBoardId}</Tag>
+          </HStack>
+        </Flex>
+        <HStack>
+          <Text fontWeight={700}>Agency: </Text>
+          <Tag>{agencyName}</Tag>
+        </HStack>
+      </VStack>
+      <HStack>
+        <Text fontWeight={700}>Policy Area: </Text>
```

Comment:
We have 700 font weight as a [token in Streetscape](https://github.com/NYCPlanning/ae-streetscape/blob/main/src/theme/tokens/font-weights.ts#L2) called `bold`.